### PR TITLE
Fix entry number example missing fields

### PR DESCRIPTION
**Pull request by acarnagey**
Updates the entry number certificate to include issuer organization location in the example.

### CODE REVIEW

**Review by mkhraisha — APPROVED** [review]

**Review by BenjaminMoe — APPROVED** [review]

**Review by OR13 — APPROVED** [review]

**Review by nissimsan — APPROVED** [review]

File: docs/openapi/components/schemas/credentials/EntryNumberCredential.yml [hunk 1]
```diff
@@ -106,7 +106,22 @@ example: |-
...
+          "Place"
+        ],
+        "address": {
+          "type": [
+            "PostalAddress"
+          ],
+          "streetAddress": "19 Knox St",
+          "addressLocality": "Toronto",
+          "addressRegion": "ON",
+          "addressCountry": "Canada",
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/credentials/EntryNumberCredential.yml:121]: Can we make this two letter code? i.e. CA

**Comment by mkhraisha** [comment]
Please merge out of band after two letter country is fixed

**Comment by mkhraisha** [comment]
@acarnagey any updates here?

File: docs/openapi/components/schemas/credentials/EntryNumberCredential.yml (hunk 1 above)
Comment (OR13) [inline docs/openapi/components/schemas/credentials/EntryNumberCredential.yml:121]:
```suggestion
          "addressCountry": "CA",
```

**Pull request merged**